src/main/java/org/manifold/compiler/front/ExpressionContextVisitor.java

```diff
@@ -315,8 +338,46 @@ private ExpressionVertex createVariableVertex(VariableIdentifier id) {
   }
 
   @Override
-  public ExpressionVertex visitImportStatement(ImportStatementContext context) {
-    return null;
+  public ExpressionVertex visitImportExpr(ImportExprContext context) {
+    String importString = context.STRING_VALUE().getText()
+        .replaceAll("\\\"", "\"");
+    importString = importString.substring(1, importString.length() - 1);
+
+    // Reassign to a new variable to satisfy the lambda requirement of "final" variables
+    String filePath = importString;
+
+    File importedFile = Stream.of(
+        new File(inputFile.getParent(), filePath),
+        new File(inputFile.getParent(), filePath + ".manifold")
+    ).filter(f -> f.exists())
+        .findFirst()
+        .orElseGet(() -> {
+          File lib = getLib(filePath);
+          if (lib == null) {
+            lib = getLib(filePath + ".manifold");
+          }
+          if (lib != null) {
+            return lib;
+          }
+          errors.add("Import " + filePath + " not found");
+          return null;
+        });
+    if (importedFile == null) {
+      return null;
+    }
+
+    ExpressionGraph g;
+    try {
+      g = parser.parseFile(importedFile);
```

Comment:
I don't think this is the best place to put this, but I'm not really sure where it would make the most sense. The alternative that seems to make the most sense is during elaboration of the `ImportVertex`.